### PR TITLE
Restructure how tos

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Restructured the How-tos to make them more agnostic and follow the SDK structure.

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work

### CODE REVIEW

**Comment by Dr-Electron** [comment]
Preview: https://iota-wiki-d9gnijuzn-iota1.vercel.app

File: docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx [hunk 1]
```diff
@@ -16,6 +16,12 @@ import AboutAccounts from '../../../_admonitions/_about-accounts.md';
 
 <AboutAccounts/>
 
+:::info
+
+Be aware that SMR has 6 decimals the L2 representation 18. So if you are don't send enough fractions of a Shimmer to L1 the transation will fail.
```
Comment (jlvandenhout) [inline docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx:21]: Please don't try to write sentences at midnight, it seems to affect your understanding of how languages work :trollface:

Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx:21]: Yup, it's not very clear, how about:
```suggestion
ShimmerEVM has 18 decimal places, while Shimmer has 6. This means that any decimals beyond the 6th will be ignored by Shimmer, even though you can see them on ShimmerEVM. Please keep this in mind while sending tokens. 
```

**Review by lucas-tortora — COMMENTED** [review]
The new order in the sidebar looks good. However, we need to rephrase the 18 vs 6 decimals admonition. I've left a suggestion. Let me know if it doesn't quite work and we can workshop something on slack and close this :)

File: docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx (hunk 1 above)
Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx:21]:
```suggestion
ShimmerEVM has 18 decimal places, while Shimmer has 6. This means that any decimals beyond the 6th will be ignored by Shimmer, even though you can see them on ShimmerEVM. Please keep this in mind while sending tokens, as your transaction will fail if its value is less than 1 glow or what is needed to cover its [storage deposit](/learn/protocols/stardust/core-concepts/storage-deposit/).
```

Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx:21]:
```suggestion
ShimmerEVM has 18 decimal places, while Shimmer has 6. This means that any decimals beyond the 6th will be ignored by Shimmer, even though you can see them on ShimmerEVM. Please keep this in mind while sending your tokens to L1.
```

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**